test/commands/wait.js

```diff
@@ -12,8 +12,8 @@ const all = [
 ]
 const fetcher = () => {
   return {
-    all: () => all,
-    addon: () => all[0]
+    all: () => Promise.resolve(all),
+    addon: () => Promise.resolve(all[0])
```

Comment:
These changes ensure that the tests (correctly) fail without the code changes above.